### PR TITLE
refactor Glyph to FontIcon usage & remove SVG/Batik dependencies

### PR DESCRIPTION
**Pull request by RalphSteinhagen**
move from ControlsFX's [Glyph](https://github.com/controlsfx/controlsfx/issues/1195) to the [Ikonli](https://github.com/kordamp/ikonli) FontIcon implementation.

N.B. Ikonli includes FontAwesome(5) and provides a more generalized and maintainable interface also for other icon fonts. 

Addresses ChartFX roadmap item #44

### CODE REVIEW

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
+@font-face {
+  font-family: 'fair-chart-icons';
+  src:  url('fonts/fair-chart-icons.eot?yr8ymj');
+  src:  url('fonts/fair-chart-icons.eot?yr8ymj#iefix') format('embedded-opentype'),
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:4]: Codacy found an issue: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

**Comment by RalphSteinhagen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 24
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990&bid=18795422)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  content: "\45";
+  color: rgb(237, 28, 36);
+}
+.icon-error_icon .path2:before {
+  content: "\46";
+  margin-left: -1em;
+  color: rgb(255, 255, 255);
+}
+
+
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:56]: Codacy found an issue: [Expected no more than 1 empty line (max-empty-lines)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+.icon-warn_icon .path2:before {
+  content: "\58";
+  margin-left: -1.1376953125em;
+  color: rgb(0, 0, 0);
+}
+.icon-error_icon .path1:before {
+  content: "\45";
+  color: rgb(237, 28, 36);
+}
+.icon-error_icon .path2:before {
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:50]: Codacy found an issue: [Expected double colon pseudo-element notation (selector-pseudo-element-colon-notation)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    url('fonts/fair-chart-icons.woff?yr8ymj') format('woff'),
+    url('fonts/fair-chart-icons.svg?yr8ymj#fair-chart-icons') format('svg');
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: 'fair-chart-icons' !important;
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:15]: Codacy found an issue: [Use of !important](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/java/de/gsi/chart/plugins/EditAxis.java [hunk 5]
```diff
@@ -359,7 +355,7 @@ private Pane getAutoRangeCheckBoxes(final Axis axis) {
             return boxMax;
         }
 
-        private final TextField getBoundField(final Axis axis, final boolean isLowerBound) {
+        private TextField getBoundField(final Axis axis, final boolean isLowerBound) {
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/java/de/gsi/chart/plugins/EditAxis.java:358]: Codacy found an issue: [The method 'getBoundField(Axis, boolean)' has an NPath complexity of 320](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/java/de/gsi/chart/renderer/spi/utils/ChartIconFactory.java [hunk 6]
```diff
@@ -1,63 +1,84 @@
...
-    private static final int MAX_HEIGHT = 100;
+public final class ChartIconFactory { // NOPMD - nomen est omen
+
+    private static final String ICON_INFO = "I";
+    private static final String ICON_WARN = "W";
+    private static final String ICON_ERROR = "E";
+    private static final Map<String, Color[]> colourMap = new HashMap<>();
 
+    private static final int DEFAULT_HEIGHT = 32;
+    private static Font iconFont;
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/java/de/gsi/chart/renderer/spi/utils/ChartIconFactory.java:24]: Codacy found an issue: [Avoid unused private fields such as 'iconFont'.](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+}
+.icon-error_icon .path2:before {
+  content: "\46";
+  margin-left: -1em;
+  color: rgb(255, 255, 255);
+}
+
+
+.text {
+    -fx-font-family: "fair-chart-icons";
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:58]: Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.icon-info_icon .path1:before {
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:28]: Codacy found an issue: [Expected double colon pseudo-element notation (selector-pseudo-element-colon-notation)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css (hunk 3 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:50]: Codacy found an issue: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+  content: "\49";
+  color: rgb(0, 0, 205);
+}
+.icon-info_icon .path2:before {
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:32]: Codacy found an issue: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+  content: "\4a";
+  margin-left: -1em;
+  color: rgb(255, 255, 255);
+}
+.icon-warn_icon .path1:before {
+  content: "\57";
+  color: rgb(255, 215, 0);
+}
+.icon-warn_icon .path2:before {
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:41]: Codacy found an issue: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/java/de/gsi/chart/renderer/spi/AbstractMetaDataRendererParameter.java [hunk 11]
```diff
@@ -2,16 +2,18 @@
...
 import javafx.beans.property.DoubleProperty;
 import javafx.beans.property.SimpleBooleanProperty;
 import javafx.beans.property.SimpleDoubleProperty;
 import javafx.beans.property.SimpleStringProperty;
 import javafx.beans.property.StringProperty;
-import javafx.scene.image.Image;
+import javafx.scene.Node;
 import javafx.scene.paint.Color;
 import javafx.scene.paint.Paint;
+import javafx.scene.text.Text;
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/java/de/gsi/chart/renderer/spi/AbstractMetaDataRendererParameter.java:14]: Codacy found an issue: [Avoid unused imports such as 'javafx.scene.text.Text'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

Comment (RalphSteinhagen) [inline chartfx-chart/src/main/java/de/gsi/chart/renderer/spi/AbstractMetaDataRendererParameter.java:14]: Codacy found an issue: [Unused import - javafx.scene.text.Text.](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+  content: "\58";
+  margin-left: -1.1376953125em;
+  color: rgb(0, 0, 0);
+}
+.icon-error_icon .path1:before {
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:46]: Codacy found an issue: [Expected double colon pseudo-element notation (selector-pseudo-element-colon-notation)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css [hunk 13]
```diff
@@ -0,0 +1,59 @@
+@font-face {
+  font-family: 'fair-chart-icons';
+  src:  url('fonts/fair-chart-icons.eot?yr8ymj');
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:3]: Codacy found an issue: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/test/java/de/gsi/chart/renderer/spi/ErrorDataSetRendererTests.java [hunk 14]
```diff
@@ -74,15 +75,15 @@ public void start(Stage stage) {
...
+    public void testRendererNominal(final LineStyle lineStyle) throws Exception {
         for (ErrorStyle eStyle : ErrorStyle.values()) {
             renderer.setErrorType(eStyle);
             testRenderer(lineStyle);
         }
     }
 
     @Test
-    public void testRendererSepcialCases() throws IOException, Exception {
+    public void testRendererSepcialCases() throws Exception {
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/test/java/de/gsi/chart/renderer/spi/ErrorDataSetRendererTests.java:86]: Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css (hunk 4 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:15]: Codacy found an issue: [Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+.icon-info_icon .path1:before {
+  content: "\49";
+  color: rgb(0, 0, 205);
+}
+.icon-info_icon .path2:before {
+  content: "\4a";
+  margin-left: -1em;
+  color: rgb(255, 255, 255);
+}
+.icon-warn_icon .path1:before {
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:37]: Codacy found an issue: [Expected double colon pseudo-element notation (selector-pseudo-element-colon-notation)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+  src:  url('fonts/fair-chart-icons.eot?yr8ymj#iefix') format('embedded-opentype'),
+    url('fonts/fair-chart-icons.ttf?yr8ymj') format('truetype'),
+    url('fonts/fair-chart-icons.woff?yr8ymj') format('woff'),
+    url('fonts/fair-chart-icons.svg?yr8ymj#fair-chart-icons') format('svg');
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+[class^="icon-"], [class*=" icon-"] {
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:13]: Codacy found an issue: [Expected newline after "," (selector-list-comma-newline-after)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/test/java/de/gsi/chart/plugins/EditAxisTests.java [hunk 17]
```diff
@@ -1,7 +1,6 @@
 package de.gsi.chart.plugins;
 
-import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/test/java/de/gsi/chart/plugins/EditAxisTests.java:3]: Codacy found an issue: [Avoid unused imports such as 'org.junit.jupiter.api.Assertions'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css (hunk 15 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:37]: Codacy found an issue: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css (hunk 9 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:32]: Codacy found an issue: [Expected double colon pseudo-element notation (selector-pseudo-element-colon-notation)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css (hunk 12 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:46]: Codacy found an issue: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css (hunk 10 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:41]: Codacy found an issue: [Expected double colon pseudo-element notation (selector-pseudo-element-colon-notation)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

**Comment by RalphSteinhagen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990&bid=18795422)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css [hunk 18]
```diff
@@ -0,0 +1,64 @@
...
+    url('fonts/fair-chart-icons.svg?yr8ymj#fair-chart-icons') format('svg');
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+[class^="icon-"],
+[class*=" icon-"] {
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: 'fair-chart-icons' !important;
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:16]: Codacy found an issue: [Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/java/de/gsi/chart/plugins/EditAxis.java (hunk 5 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/java/de/gsi/chart/plugins/EditAxis.java:358]: Codacy found an issue: [The method 'getBoundField(Axis, boolean)' has an NPath complexity of 320](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css [hunk 19]
```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: 'fair-chart-icons';
+  src: url('fonts/fair-chart-icons.eot?yr8ymj');
+  src: url('fonts/fair-chart-icons.eot?yr8ymj#iefix') format('embedded-opentype'),
```
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:4]: Codacy found an issue: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css (hunk 18 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:16]: Codacy found an issue: [Use of !important](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/test/java/de/gsi/chart/plugins/EditAxisTests.java (hunk 17 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/test/java/de/gsi/chart/plugins/EditAxisTests.java:3]: Codacy found an issue: [Avoid unused imports such as 'org.junit.jupiter.api.Assertions'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/test/java/de/gsi/chart/renderer/spi/ErrorDataSetRendererTests.java (hunk 14 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/test/java/de/gsi/chart/renderer/spi/ErrorDataSetRendererTests.java:86]: Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

**Comment by RalphSteinhagen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990&bid=18795422)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css (hunk 18 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:16]: Codacy found an issue: [Use of !important](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/test/java/de/gsi/chart/plugins/EditAxisTests.java (hunk 17 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/test/java/de/gsi/chart/plugins/EditAxisTests.java:3]: Codacy found an issue: [Avoid unused imports such as 'org.junit.jupiter.api.Assertions'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/java/de/gsi/chart/plugins/EditAxis.java (hunk 5 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/java/de/gsi/chart/plugins/EditAxis.java:358]: Codacy found an issue: [The method 'getBoundField(Axis, boolean)' has an NPath complexity of 320](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/test/java/de/gsi/chart/renderer/spi/ErrorDataSetRendererTests.java (hunk 14 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/test/java/de/gsi/chart/renderer/spi/ErrorDataSetRendererTests.java:86]: Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css (hunk 19 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:4]: Codacy found an issue: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css (hunk 18 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:16]: Codacy found an issue: [Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/test/java/de/gsi/chart/plugins/EditAxisTests.java (hunk 17 above)
Comment (wirew0rm) [reply chartfx-chart/src/test/java/de/gsi/chart/plugins/EditAxisTests.java:3]: This is a wrong positive pmd bug: pmd/pmd#2016

**Comment by RalphSteinhagen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990&bid=18795422)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css (hunk 18 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:16]: Codacy found an issue: [Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/test/java/de/gsi/chart/renderer/spi/ErrorDataSetRendererTests.java (hunk 14 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/test/java/de/gsi/chart/renderer/spi/ErrorDataSetRendererTests.java:86]: Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/test/java/de/gsi/chart/plugins/EditAxisTests.java (hunk 17 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/test/java/de/gsi/chart/plugins/EditAxisTests.java:3]: Codacy found an issue: [Avoid unused imports such as 'org.junit.jupiter.api.Assertions'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css (hunk 19 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:4]: Codacy found an issue: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css (hunk 18 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css:16]: Codacy found an issue: [Use of !important](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

File: chartfx-chart/src/main/java/de/gsi/chart/plugins/EditAxis.java (hunk 5 above)
Comment (RalphSteinhagen) [inline chartfx-chart/src/main/java/de/gsi/chart/plugins/EditAxis.java:358]: Codacy found an issue: [The method 'getBoundField(Axis, boolean)' has an NPath complexity of 320](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)

**Pull request merged**